pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
Here's an example of a package that patches this with a harcoded value: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/science/misc/root/sw_vers.patch